.travis.yml

```diff
@@ -0,0 +1,84 @@
+  #- sudo apt-get -y install libgeos-3.5.0=3.5.0-1cdb2
+
+  # Install postgres db and build deps
+  - sudo /etc/init.d/postgresql stop # stop travis default instance
+  - sudo apt-get -y remove --purge postgresql-9.1
+  - sudo apt-get -y remove --purge postgresql-9.2
+  - sudo apt-get -y remove --purge postgresql-9.3
+  - sudo apt-get -y remove --purge postgresql-9.4
+  - sudo apt-get -y remove --purge postgis
+  - sudo apt-get -y autoremove
+
+  - sudo apt-get -y install postgresql-9.5=9.5.2-2ubuntu1
+  - sudo apt-get -y install postgresql-server-dev-9.5=9.5.2-2ubuntu1
+  - sudo apt-get -y install odbcinst
+  - sudo apt-get -y install odbc-postgresql
+  #- sudo apt-get -y install postgresql-9.5-postgis-2.2=2.2.2.0-cdb2
+  #- sudo apt-get -y install postgresql-9.5-postgis-scripts=2.2.2.0-cdb2
+
+  # configure it to accept local connections from postgres
+  - echo -e "# TYPE  DATABASE        USER            ADDRESS                 METHOD \nlocal   all             postgres                                trust\nlocal   all             all                                     trust\nhost    all             all             127.0.0.1/32            trust" \
+    | sudo tee /etc/postgresql/9.5/main/pg_hba.conf
+  - sudo /etc/init.d/postgresql restart 9.5
+
+  # Local MySQL
+  - sudo apt-get -y install mysql-server-5.5
+  - sudo apt-get -y install libmyodbc
+
+  - sudo cat /etc/odbcinst.ini
+
+  - echo "[PostgreSQL ANSI]" | sudo tee /etc/odbcinst.ini
+  - echo "Description = PostgreSQL ODBC driver (ANSI version)" | sudo tee -a /etc/odbcinst.ini
+  - echo "Driver      = psqlodbca.so" | sudo tee -a /etc/odbcinst.ini
+  - echo "Setup       = libodbcpsqlS.so" | sudo tee -a /etc/odbcinst.ini
+  - echo "Debug       = 0" | sudo tee -a /etc/odbcinst.ini
+  - echo "CommLog     = 1" | sudo tee -a /etc/odbcinst.ini
+  - echo "UsageCount  = 1" | sudo tee -a /etc/odbcinst.ini
+  - echo "\n" | sudo tee -a /etc/odbcinst.ini
+  - echo "[PostgreSQL Unicode]" | sudo tee -a /etc/odbcinst.ini
+  - echo "Description = PostgreSQL ODBC driver (Unicode version)" | sudo tee -a /etc/odbcinst.ini
+  - echo "Driver      = psqlodbcw.so" | sudo tee -a /etc/odbcinst.ini
+  - echo "Setup       = libodbcpsqlS.so" | sudo tee -a /etc/odbcinst.ini
+  - echo "Debug       = 0" | sudo tee -a /etc/odbcinst.ini
+  - echo "CommLog     = 1" | sudo tee -a /etc/odbcinst.ini
+  - echo "UsageCount  = 1" | sudo tee -a /etc/odbcinst.ini
+  - echo "\n" | sudo tee -a /etc/odbcinst.ini
+  - echo "[MySQL]" | sudo tee -a /etc/odbcinst.ini
+  - echo "Description = MySQL driver" | sudo tee -a /etc/odbcinst.ini
+  - echo "Driver      = libmyodbc.so" | sudo tee -a /etc/odbcinst.ini
+  - echo "Setup       = libodbcmyS.so" | sudo tee -a /etc/odbcinst.ini
+  - echo "CPTimeout   =" | sudo tee -a /etc/odbcinst.ini
+  - echo "CPReuse     =" | sudo tee -a /etc/odbcinst.ini
+  - echo "UsageCount  = 1" | sudo tee -a /etc/odbcinst.ini
```

Comment:
Point by point:

- Have it inside Travis is ugly but we have it bounded for this kind of infrastructure (Travis). In case we don't want to use Travis, this file is not needed anymore. Is part of the configuration of the connector.
- That file is not needed for the tests to work but the config file are needed to:
  - Generate the installation tests
  - Be able to test and connect to the connectors